source/features/dim-bots.tsx

```diff
@@ -15,16 +15,18 @@ const commitSelectors = [
 	'scala-steward',
 	'snyk-bot',
 	'web-flow',
-].map(bot => `.commit-author[href$="?author=${bot}"]`);
-
-commitSelectors.push('.commit-author[href$="%5Bbot%5D"]'); // Generic [bot] label
+];
 
+const commitSelectors = botNames.map(bot => `.commit-author[href$="?author=${bot}"]`);
+commitSelectors.push('.commit-author[href$="%5Bbot%5D"]'); // Generic `[bot]` label in author name
 const commitSelector = commitSelectors.join(',');
 
-const prSelector = [
-	'.opened-by [href*="author%3Aapp%2F"]',
-	'.labels [href$="label%3Abot"]',
-];
+const prSelectors = botNames.map(bot => `.opened-by [title="pull requests opened by ${bot}"]`);
```

Comment:
This allows re-using the list of bot names to also hide potential PRs opened by them. 

I'm guessing not all of them can do that, but their usernames are reserved anyway so we don't risk hiding legitimate PRs.